src/PDB_ModuleInfoStream.cpp

```diff
@@ -137,7 +137,7 @@ PDB::ModuleInfoStream::ModuleInfoStream(const DirectMSFStream& directStream, uin
 		streamOffset += objectNameLength + 1u;
 
 		// the stream is aligned to 4 bytes
-		streamOffset = BitUtil::RoundUpToMultiple(streamOffset, 4ull);
+		streamOffset = BitUtil::RoundUpToMultiple(streamOffset, static_cast<size_t>(4ul));

```

Comment:
I prefer using an explicit template argument in that case, i.e. BitUtil::RoundUpToMultiple<size_t>(arg1, arg2).